api_proxy/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# How to build:
+# docker run --rm -e BUILD_GOOS="linux" -e BUILD_GOARCH="amd64" -v $(pwd):/src \
+#   -v /var/run/docker.sock:/var/run/docker.sock \
+#   centurylink/golang-builder-cross gcr.io/$DOCKER_NAMESPACE/api-proxy
```

Comment:
What's this for? We don't need a cross builder for targeting amd64 linux, that's what you're building in already.

Maybe you want something like go2docker, which I fixed recently: https://github.com/kubernetes/contrib/tree/master/go2docker